ui/apps/ui/src/app/collections/data/all/adapter.data.ts

```diff
@@ -71,6 +71,31 @@ const forInteroperabilityGuidelinesValueAdapter = (value: string = '') => {
     : value;
 };
 
+const getDate = (
```

Comment:
I would suggest changing it to function (instaad of lambda) with return typing. Also - a name like `extractDate` would be more descriptive.